### PR TITLE
Analytics hub product name can exceed the area

### LINKED ISSUE #7812: [Analytics Hub] Product name can exceed the area
**Describe the bug**
When having a long product name, the label will exceed its limits. We should ellipsis (or use 2 rows?) it in such case.

**Screenshots**
Screenshot 2022-11-11 at 10 21 08

### LINKED ISSUE #8030: [Analytics Hub] Top Performers product list bottom spacing is off
## Summary
Analytics Hub in top performers list last row height is less than expected, feels cut off (but no content is cut)

## Screenshot
image

### PR DESCRIPTION
**Pull request by kidinov**
Closes: #7812
Closes: #8030

### Description

* Adds placeholder for the products on the Analytics screen
* Improves layout so long text won't break that 

### Testing instructions

* Open analytics screen with different products on it
* Notice that it looks correct now

#### Before:
image

#### After:

image

- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

**Comment by hichamboushaba** [comment]
@kidinov for the instrumented-tests failure above, it was fixed in `trunk` today, so merge it and it should fix it.

File: RELEASE-NOTES.txt [hunk 1]
```diff
@@ -1,6 +1,7 @@
 *** PLEASE FOLLOW THIS FORMAT: [<priority indicator, more stars = higher priority>] <description> [<PR URL>]
 11.8
 - [*] Add variations bottom sheet is fully visible right after showing up in the landscape mode [https://github.com/woocommerce/woocommerce-android/pull/8043]
+- [***] UI improvements to the products on the analytics screen [https://github.com/woocommerce/woocommerce-android/pull/8098]
```
Comment (wzieba) [inline RELEASE-NOTES.txt]: Nitpick, but maybe it'll be helpful for someone providing release notes: I'm not sure if this improvement (although really appreciated) is that important 😅 .
```suggestion
- [*] UI improvements to the products on the analytics screen [https://github.com/woocommerce/woocommerce-android/pull/8098]
```

**Review by wzieba — APPROVED** [review]
Thank you for taking care of this, looks neat! Leaving one super minor but feel free to merge the PR if you think otherwise 👍

**Comment by wzieba** [comment]
~But I'm not sure if it closes #8030 , does it? The bottom space is still very close to the screen edge.~

Sorry, I didn't get the description of #8030 right. Looks good 👍

File: RELEASE-NOTES.txt (hunk 1 above)
Comment (kidinov) [reply RELEASE-NOTES.txt]: I got confused - somehow I was thinking that more starts - lower priority at the time of writing this. Thank you for notificing

Comment (wzieba) [reply RELEASE-NOTES.txt]: Some time ago I had *exactly* the same confusion: https://github.com/woocommerce/woocommerce-android/pull/8020#discussion_r1049989295 😅 Thanks for the update!

**Pull request merged**